Makefile

```diff
@@ -0,0 +1,20 @@
+DIRECTORIES?=dimension service
+
+BUILD=build
+HASH?=$(shell make hash)
+BUILD_ARCH=$(BUILD)/$(GOOS)-$(GOARCH)
+BIN_DIR?=.
+
+export GOOS?=$(shell go env GOOS)
+export GOARCH?=$(shell go env GOARCH)
+
+build:
+	@mkdir -p $(BUILD_ARCH)/$(BIN_DIR)
+	go build -o $(BUILD_ARCH)/$(BIN_DIR)/dp-dimension-extractor main.go
+
+debug:
+	HUMAN_LOG=1 go run main.go
+
+test: ; $(foreach dir,$(DIRECTORIES),(go test -cover $(dir)/*.go) &&) :
```

Comment:
When the tests are ran can you echo the directory name out. It helps which directory is being ran on the CI server